apps/test/solutions/applab/ec_canvas.js

```diff
@@ -191,3 +185,12 @@ module.exports = {
     }
   ]
 };
+
+var canvasAndResultsPopulated = function () {
```

Comment:
I think our style guide says top level functions should be in the form
```
function canvasAndResultsPopulate...
```
Though I don't care too much in this case.